File: dotnet-test-samples/apigw-lambda-ddb/tests/ApiTests.IntegrationTest/Drivers/Product.cs
namespace ApiTests.IntegrationTest.Drivers
{
    public class Product
    {
        public Product(string id, string name, decimal price)
        {
            Id = id;
            Name = name;
            Price = price;
        }
        
        public string Id { get; set; }
        
        public string Name { get; set; }
        
        public decimal Price { get; set; }
    }
}

File: typescript-test-samples/typescript-sample-app/src/model/Product.ts
// Copyright Amazon.com, Inc. or its affiliates. All Rights Reserved.
// SPDX-License-Identifier: MIT-0

export interface Product {
  id: string;
  name: string;
  price: number;
}

export interface MetaProduct extends Product {
  timeCreated?: string;
  timeQueued?: string;
  timeStored?: string;
}

export type AnyProduct = Product | MetaProduct;
